addons/interaction/functions/fnc_start_desk_close.sqf

```diff
@@ -0,0 +1,23 @@
+/**
+ * Starts process of closing a desk
+ *
+ * Arguments:
+ * 0: Desk <OBJECT>
+ *
+ * Return Value:
+ * None
```

Comment:
The function should return a bool, like the function you wrapped around. We already use a wrapper that needs the return value.